docs/pages/contributing/documentation/style-guide.mdx

```diff
@@ -1,9 +1,34 @@
 ---
-title: Style Guide
-description: Use this reference to ensure our docs are consistent and effective
+title: Style guide
+description: Provides guidelines, documentation conventions, and recommended practices for contributing to Teleport documentation.
 ---
 
-To keep our documentation effective as Teleport develops, we need to ensure that we maintain a consistent approach to organizing information.
+Technical documentation can be a challenging business, and this challenge can mean 
+wrangling a many-headed, many-handed beast. To help us stay on course—provide our 
+North Star—this guide describes our approach to building effective documentation, 
```

Comment:
```suggestion
This guide describes our approach to building effective documentation, 
```

I think we can make this less figurative, since our audience doesn't necessarily want to know what it's like to write docs.